data/threatbook

```diff
@@ -0,0 +1 @@
+threatbook.cn
```

Comment:
If you are trying to whitelist this domain, geolocation-cn should have covered it.
If you are including this domain for some different use-case, then I must know that before I can approve this PR.

Reply:
I don't see where the geolocation-cn file is located. If I just want to customize the domain access how do I set it up?

Reply:
> I just checked geolocation-cn and found that there is no such domain inside, so I want to add it

`tld-cn` will cover all `.cn` domains.

Reply:
Understood, the problem is solved. How do I add a domain name for my personal needs website? If I encounter a domain that does not exist inside geolocation-cn, is it possible to pr submit it again?

Reply:
> How do I add a domain name for my personal needs website?

You can generate your own geosite.dat file. https://github.com/Loyalsoldier/v2ray-rules-dat is a good place to start.

---

> If I encounter a domain that does not exist inside `geolocation-cn`, is it possible to pr submit it again?

Sure. Just issue another PR and wait for someone to review it ;-)